### ISSUE TITLE
Instabot doesn't count well!

### ISSUE THREAD

**Issue by Github1010** [instabot-py/instabot.py#455]
Instabot is "liking" the media, but I don't know why, it doesn't count well!
You can see that maybe says:
'like NUMBER 295 and the next is LIKE NUMBER 290'
Do you know why?
Because in 8h my instagram has followed 600 persons and instabot says only 300!
THANK YOU

![error like](https://cloud.githubusercontent.com/assets/26791177/24601441/35807d68-1859-11e7-91db-7c8ede0c2d95.jpg)

**Comment by LevPasha** [comment]
O_o maybe you run 2 bot process? And they write all on your console?

**Comment by ChipWolf** [comment]
I don't see how this would be possible, there's nothing that could count down, I think @LevPasha could be right.

**Comment by nicholaai** [comment]
I noticed that in Mode 2 the counting is off as well. I was only running one instance and the like count would be off. For example would say like number 122, to like number 157 for the first few likes. I noticed @ChipWolf said most modes are unstable and he just uses mode 0 so I went back to that mode. What mode are you in @Github1010 ?

**Comment by Github1010** [comment]
I was using mode 0


Joan

-------- Mensaje original --------
De: Nicholas Pappagalo <notifications@github.com> 
Fecha: 03/04/2017  18:08  (GMT+01:00) 
Para: "instabot-py/instabot.py" <instabot.py@noreply.github.com> 
Cc: Github1010 <joan1593@gmail.com>,Mention <mention@noreply.github.com> 
Asunto: Re: [instabot-py/instabot.py] Instabot doesn't count well! (#455) 
 
I noticed that in Mode 2 the counting is off as well. I was only running one instance and the like count would be off. For example would say like number 122, to like number 157 for the first few likes. I noticed @ChipWolf said most modes are unstable and he just uses mode 0 so I went back to that mode. What mode are you in @Github1010 ?
